.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1776,6 +1776,68 @@ stages:
         command: "CheckBuildLogsForErrors"
         apiKey: $(DD_LOGGER_DD_API_KEY)
 
+- stage: profiler_integration_tests_windows
```

Comment:
I think we should only be running this and the linux integration tests if there are changes to the profiler? So we should probably add this to the condition:

```
eq(dependencies.generate_variables.outputs['generate_variables_job.generate_variables_step.IsProfilerChanged'], 'True'),
```

Reply:
As of today, tests that check communication between the profiler and the tracer are in the profiler.
So if we change something in the tracer (CodeHotSpot for example) it would be nice to catch.